content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -158,6 +158,41 @@ and
 Note that the information reported as Pod status depends on the current
 [ContainerState](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
 
+## Container States
+
+Once pod is assigned to a node by scheduler, kubelet starts creating container using container runtime.There are three possible states of containers, i.e Waiting, Running and Terminated. To check state of container, you can use `kubectl describe pod` command. State will be displayed for each container within that pod.
+
+* `Waiting`: This is default state of container. If container is not in running or terminated state, it is in waiting state. In this        state container is still creating and doing all required things like pulling images, applying secrets etc.
+   Along with this state, message and reason about this state is also displayed to give more information about waiting state.
+
+    ```yaml
+   ...
+      State:          Waiting
+       Reason:       ErrImagePull
+	  ...
+   ```
+   
+* `Running`: Running state indicate that container is executing without any issue. Once container enters into running state, `postStart`    hook (if any) will be executed. This state also displays the time from which it is in running state i.e.start time.  
```

Comment:
Change to:

"* `Running`: Indicates that the container is executing without issues. Once a container enters into Running, `postStart` hook (if any) is executed. This state also displays the time when the container entered Running state:"

Reply:
Suggested changes are committed. Please check